capsul/plugins/__init__.py

```diff
@@ -11,5 +11,5 @@
     ("caps", "http://nsap.intra.cea.fr/caps-doc/"),
     ("qmri", "https://bioproj.extra.cea.fr/redmine/projects/qmri/publishing/"),
     ("funtk", "https://bioproj.extra.cea.fr/redmine/projects/funtk/publishing/"),
-    ("pclinfmri", "")
```

Comment:
There is no documentation here. I do not know (or remember) how this module is used.